.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build package
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        python setup.py sdist bdist_wheel
+
+    - name: Upload artifacts
+      if: ${{ success() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist/*.whl
+
+    - name: Set up venv
+      run: |
+        sudo apt-get install python3-venv
+        python -m venv ~/myenv
+        source ~/myenv/bin/activate
+        echo "Virtual env: $VIRTUAL_ENV"
+
+    - name: Install pytest and our package
+      run: |
+        pip install pytest
+        cd ..
+        pip install -f pydicom/dist pydicom
+
+    - name: Test with pytest
+      run: |
+        python -c "import pytest; pytest.main(['--pyargs', 'pydicom.tests'])"
```

Comment:
Do you want to run the full test suite here? Numpy, etc?

Reply:
Well, all tests would have been run on the push to master leading up to this (all python, all variations), so this is really just to check that the build has not somehow gone wrong.  That could happen if built on a local machine, where there could be extra files not in git, but it shouldn't be possible in a fresh build from newly checked out version, unless somehow some files were not included in the build.

Do you see any potential for problems?  I suppose at least ensuring numpy could be prudent.  I'm not too keen on trying to replicate the whole test matrix here.  Perhaps a different way to get there would be to modify the 'push to master' workflow to build 'dist' and test from the 'pip install'd build?

Reply:
> Do you see any potential for problems?

Not really, just wanted to double check